stable/kubernetes-dashboard/templates/ingress.yaml

```diff
@@ -1,6 +1,7 @@
 {{- if .Values.ingress.enabled -}}
 {{- $serviceName := include "kubernetes-dashboard.fullname" . -}}
 {{- $servicePort := .Values.service.externalPort -}}
+{{- $path := default "/" .Values.ingress.path -}}
```

Comment:
Instead of using `default` you could un-comment the default value in the `values.yaml` file and using `.Values.ingress.path` in place of `$path` in the template. Is there a reason to use `default`?

Reply:
Nope, I can do that. The only reason I used default instead was to keep the style the same with the rest of the ingress fields in `values.yaml`.